dev/tools/build-ce.php

```diff
@@ -0,0 +1,179 @@
+<?php
```

Comment:
Do we have cases when we need this for standalone installation? IF no, please delete it